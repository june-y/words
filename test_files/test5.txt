seal,fog
L	G	N	S	Q
U	I	E	K	E
Z	A	O	N	M
L	F	O	G	J
S	I	O	A	L

