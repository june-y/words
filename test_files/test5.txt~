seal,fog
L	G	N	S	Q
U	I	E	K	E
Z	A	O	N	M
L	F	O	G	J
S	I	O	A	L



[["s", [0, 3]], ["e", [1, 4]], ["e", [1, 2]], ["a", [2, 1]], ["l", [3, 0]], ["l", [4, 4]]]


[0,3], [1, 2] "a", [2, 1] l [4, 4]


distance = square_root[(x2 -x1) + (y2-y1)]


