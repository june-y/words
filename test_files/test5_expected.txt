seal: (0,3),(1,2),(2,1),(3,0)
fog: (3,1),(3,2),(3,3)
L	G	N	S	Q
U	I	E	K	E
Z	A	O	N	M
L	F	O	G	J
S	I	O	A	L

